### ISSUE TITLE
Transaction history / log

### ISSUE THREAD

**Issue by gordsport** [vegaprotocol/vegawallet-desktop#237]
As a user I want to see (either the dApp or the Wallet(**preferred**)) the transaction history. I also wish to have links we should have links to the BlockExplorer to find more info.

Discussion notes:
Scope with no Core support: Is this works in the current session. (highlight TXs that failed/error/rejected/spam etc...)
Work around: you can add you key into block explorer for old transactions

SCOPE INFO: Core/Wallet support for history older than current session TXs is Out of Scope if this is not yet supported by core, or cannot be implemented in time.

**Comment by ValentinTrinque** [comment]
An API has been added to see all the transactions sent since the start up of the software. As a result, when closed, all the history disappears. Normally, we return what's needed to point to the block explorer.

**Comment by ValentinTrinque** [comment]
See:
- #218

**Comment by campbellssource** [comment]
See the different transaction types in the [design system ](https://www.figma.com/file/F995ESZ0QMpTBuQWGAgLRz?node-id=501:18725)

**Comment by gordsport** [comment]
Blocked until we can have full transaction history - do we still need to do this before trading is enabled?

**Comment by campbellssource** [comment]
See also the acceptance criteria in https://github.com/vegaprotocol/specs/pull/1181 specifically in WALL headings #transaction and #transactions

**Comment by campbellssource** [comment]
For a bit more context here is where the list of transactions could live in the app. 
on the selected key page.

<img width="1823" alt="Screenshot 2022-08-17 at 09 49 23" src="https://user-images.githubusercontent.com/13255539/185076528-f1c45ffa-479e-413d-bed9-52796e2db07a.png">
full [Figma](https://www.figma.com/file/W8SV56AAtXpRoQJDQW9ZYt/Vega-Wallet?node-id=1317%3A18325)

**Comment by pennyandrews** [comment]
Agreed in FE backlog blockers call that the current possible scope to show current session is sufficient and valuable.

**Comment by notbot00** [comment]
currently blocked by https://github.com/vegaprotocol/vegawallet-desktop/pull/307, then potentially need some backend work to make sure there's useful data coming through

**Comment by notbot00** [comment]
Since the scope has been refined, the pending transaction part now is working:

![Image](https://user-images.githubusercontent.com/105208209/194270641-088953c6-dd69-4027-b64c-d52dbdefa39f.png)
![Image](https://user-images.githubusercontent.com/105208209/194270642-879afe6c-7d73-4fe7-9f1a-9dc5a846b383.png)
